spec/webpack.use-network.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path')
```

Comment:
Same. Can we rename this file to `webpack.allow-from-url.html`?

Reply:
I guess you mean `webpack.allow-from-url.config.js`. Yes we can!